test/integration/template_collector_test.go

```diff
@@ -18,6 +18,8 @@ import (
 func checkSidecarContainers(t *testing.T, containers []v1.Container, options *helm.Options, chartPath string) {
 	require.NotEmpty(t, containers)
 	found := 0
+	securityContextEnabled := options.SetValues["admin.securityContext.enabledOnContainer"] == "true" ||
+		options.SetValues["database.securityContext.enabledOnContainer"] == "true"
```

Comment:
Should this be using the form that checks that the key is present, i.e. `val, ok := options.SetValues["admin.securityContext.enabledOnContainer"]`?

Reply:
The values for non-existing keys will be an empty string so this short form will work.